src/reorderable/ReorderableListComponent.coffee

```diff
@@ -0,0 +1,78 @@
+_ = require 'lodash'
+React = require 'react'
+H = React.DOM
+R = React.createElement
+DragDropContext = require('react-dnd').DragDropContext
+HTML5Backend = require('react-dnd-html5-backend')
+ReorderableListItemComponent = require "./ReorderableListItemComponent"
+
+class ReorderableListComponent extends React.Component
+  @propTypes:
+    items: React.PropTypes.array.isRequired # items to be sorted
+    onReorder: React.PropTypes.func.isRequired #callback
+    renderItem: React.PropTypes.func.isRequired
```

Comment:
Each prop needs a full enough comment that I'd know how to use it by just reading the comment